ansible/roles/tasks-systemd/cockpituous-janitor.service

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Clean up cockpituous cruft
+
+[Service]
+Type=oneshot
+# remove leftover containers
+ExecStart=/bin/sh -ec 'podman ps -q --filter status=exited | xargs -r podman rm'
+# remote old task images; this is opportunistic -- it won't remove used images, but will fail on them
+ExecStart=-/usr/bin/podman rmi --all
```

Comment:
Never seen `ExecStart=-`, is that allowed to fail syntax?

Reply:
Why not use `podman image prune`?  Should be safer and only remove unused images.

Reply:
Yes, `-` prefix means "allowed to fail", see [manpage](https://manpages.debian.org/testing/systemd/systemd.service.5.en.html).

> podman image prune

TIL! This is nicer indeed, done.

Reply:
(`--force` is terribly named -- should be `-y` or so..)